README.md

```diff
@@ -2,68 +2,25 @@
 
 This project provides common APIs for applications that live on vets.gov.
 
-## Developer Setup
-
-Vets API requires:
-- PostgreSQL
-- Redis
-- Ruby 2.3
-
 ### Base Setup
 
 To start, fetch this code:
 
 `git clone https://github.com/department-of-veterans-affairs/vets-api.git`
 
```

Comment:
I think we should check with all developers before we make docker the only documented mechanism for running vets-api. I know there were some who didn't have easy root/sudo access on various machines. I don't think anything you've done makes it impossible to use the old steps so maybe we document Docker as the preferred approach but leave the old instructions, possibly in a separate file for clarity.